Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.

In [1]:
import numpy as np
import pandas as pd
import scipy
import scipy.stats as stats

In [3]:
data=pd.read_csv(r"C:\Users\neetac1973\Downloads\BuyerRatio.csv")

In [4]:
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [5]:
data1=data.drop(['Observed Values'],axis=1)

In [6]:
data1

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [7]:
data1.mean()

East     242.5
West     832.5
North    743.5
South    410.0
dtype: float64

In [8]:
stats.f_oneway(data1.iloc[:,0], data.iloc[:,1],data.iloc[:,2])

F_onewayResult(statistic=0.6318719592523362, pvalue=0.5901945742214081)

From the above test, we get F-score = 0.6318 and p-value = 0.59019.

Since the significance value is given as 0.05, we can say that as the p-value > significance value, we failed to reject the null hypothesis.

Hence All proportions are equal.